android/src/main/java/com/rcttabview/RCTTabView.kt

```diff
@@ -45,6 +48,26 @@ class ReactBottomNavigationView(context: Context) : BottomNavigationView(context
       onTabSelected(item)
       true
     }
+    // Register long press listener for each tab
+    post {
+      for (i in 0 until menu.size()) {
+        val menuItem = menu.getItem(i)
+        findViewById<View>(menuItem.itemId).setOnLongClickListener {
+          onTabLongPressed(menuItem)
+          true
+        }
+      }
+    }
+  }
```

Comment:
I wonder what happens if you dynamically add tabs, this probably won't work for the newly added tab. Maybe this can be moved to `items` setter.

Reply:
Yeah, will move this block to updateItems function. There it will handle dynamic tabs case as well.